handler/hello.go

```diff
@@ -0,0 +1,45 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type HelloRequest struct {
+	Name string `json:"name"`
+	Path string
+}
+
+type HelloResponse struct {
+	Code      int    `json:"code"`
+	Message   string `json:"message"`
+	Timestamp string `json:"timestamp"`
+}
+
+func HelloHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
No need to repeat the package name.
```suggestion
func Hello(w http.ResponseWriter, r *http.Request) {
```